hubblestack_nova/modules/win_gp.py

```diff
@@ -208,13 +208,15 @@ def _get_tags(data):
 
 
 def _get_gp_templates():
-    domain_check = ['system.get_domain_workgroup']()
-    if 'workgroup' in domain_check:
+    domain_check = __salt__['system.get_domain_workgroup']()
+    if 'Workgroup' in domain_check:
         return False
     else:
-        domain_check = domain_check['domain']
-    list = ['cmd.run']('Get-ChildItem //domain_check/SYSVOL/domain_check | Format-List -Property '
-                                'Name, SID', shell='powershell', python_shell=True)
+        domain_check = domain_check['Domain']
+    log.error('**********')
+    log.error(domain_check)
```

Comment:
these look like leftover debug logs which should be reduced to `log.debug` or removed.